First pass at `ste_vec` docs for JSONB containment indexing (#9)
cipherstash/encrypt-query-language: README.md

However, if that same short string only appears as a part of a larger token, then it will not match that record.
In general, therefore, you should try to ensure that the string you search for is at least as long as the `tokenLength` of the index, except in the specific case where you know that there are shorter tokens to match, _and_ you are explicitly OK with not returning records that have that short string as part of a larger token.

###### ste_vec opts

> An ste_vec index on a encrypted JSONB column enables the use of Postgres's `@>` and `<@` containment operators.

Reviewer: Link to the Postgres documentation so users have more context
```suggestion
An ste_vec index on a encrypted JSONB column enables the use of PostgreSQL's `@>` and `<@` [containment operators](https://www.postgresql.org/docs/16/functions-json.html#FUNCTIONS-JSONB-OP-TABLE).
```

An ste_vec index requires one piece of configuration: the `prefix` (a string) which is functionally similar to a salt for the hashing process.

> Within a dataset, encrypted columns indexed using an ste_vec that use different prefixes can never compare as equal and containment queries that manage to mix index terms from multiple columns will never return a positive result. This is by design.

Reviewer: Line per sentence, to improve editability
```suggestion
Within a dataset, encrypted columns indexed using an ste_vec that use different prefixes can never compare as equal. 
Containment queries that manage to mix index terms from multiple columns will never return a positive result. 
This is by design.
```

The index is generated from a JSONB document by first flattening the structure of the document such that a hash can be generated for each unique path prefix to a node.

> The complete set of JSON types is supported by the indexer. Null values are ignored by the indexer.

Reviewer: Line per sentence, to improve editability

```suggestion
The complete set of JSON types is supported by the indexer. 
Null values are ignored by the indexer.
```

###### ste_vec opts

> An ste_vec index on a encrypted JSONB column enables the use of PostgreSQL's `@>` and `<@` [containment operators](https://www.postgresql.org/docs/16/functions-json.html#FUNCTIONS-JSONB-OP-TABLE).

Author: Nice
